app/static/js/events/wizard/basic-details/initializer.js

```diff
@@ -32,6 +32,7 @@ function basicDetailsInit($eventDiv) {
     $eventDiv.find(".licence-help").click(function () {
         $("#licence-list").slideToggle();
     });
+    $eventDiv.find("#ticket").val(0.01);
```

Comment:
@niranjan94  that was left unintentionally :) already have set the value in the object.